### ISSUE TITLE
error: 'database is locked'

### ISSUE THREAD

**Issue by gsemet** [morpheus65535/bazarr#150]
Hello.
With latest release I get this error after a while:
```
2018-09-23 17:55:58	stdout	Serving on http://0.0.0.0:6767
2018-09-23 21:25:03	stdout	Traceback (most recent call last):
2018-09-23 21:25:03	stdout	  File "/app/libs/bottle.py", line 862, in _handle
2018-09-23 21:25:03	stdout	    return route.call(**args)
2018-09-23 21:25:03	stdout	  File "/app/libs/bottle.py", line 1740, in wrapper
2018-09-23 21:25:03	stdout	    rv = callback(*a, **ka)
2018-09-23 21:25:03	stdout	  File "/app/bazarr.py", line 139, in wrapper
2018-09-23 21:25:03	stdout	    return func(*a, **ka)
2018-09-23 21:25:03	stdout	  File "/app/bazarr.py", line 758, in wanted_search_missing_subtitles_list
2018-09-23 21:25:03	stdout	    ref = request.environ['HTTP_REFERER']
2018-09-23 21:25:03	stdout	KeyError: 'HTTP_REFERER'
2018-09-23 21:30:53	stdout	Traceback (most recent call last):
2018-09-23 21:30:53	stdout	  File "/app/libs/bottle.py", line 862, in _handle
2018-09-23 21:30:53	stdout	    return route.call(**args)
2018-09-23 21:30:53	stdout	  File "/app/libs/bottle.py", line 1740, in wrapper
2018-09-23 21:30:53	stdout	    rv = callback(*a, **ka)
2018-09-23 21:30:53	stdout	  File "/app/bazarr.py", line 139, in wrapper
2018-09-23 21:30:53	stdout	    return func(*a, **ka)
2018-09-23 21:30:53	stdout	  File "/app/bazarr.py", line 760, in wanted_search_missing_subtitles_list
2018-09-23 21:30:53	stdout	    wanted_search_missing_subtitles()
2018-09-23 21:30:53	stdout	  File "/app/get_subtitle.py", line 468, in wanted_search_missing_subtitles
2018-09-23 21:30:53	stdout	    wanted_download_subtitles(episode[0])
2018-09-23 21:30:53	stdout	  File "/app/get_subtitle.py", line 372, in wanted_download_subtitles
2018-09-23 21:30:53	stdout	    c_db.execute('UPDATE table_episodes SET failedAttempts = ? WHERE sonarrEpisodeId = ?', (unicode(attempt), episode[2]))
2018-09-23 21:30:53	stdout	OperationalError: database is locked
2018-09-23 22:21:36	stdout	Traceback (most recent call last):
2018-09-23 22:21:36	stdout	  File "/app/libs/bottle.py", line 862, in _handle
2018-09-23 22:21:36	stdout	    return route.call(**args)
2018-09-23 22:21:36	stdout	  File "/app/libs/bottle.py", line 1740, in wrapper
2018-09-23 22:21:36	stdout	    rv = callback(*a, **ka)
2018-09-23 22:21:36	stdout	  File "/app/bazarr.py", line 139, in wrapper
2018-09-23 22:21:36	stdout	    return func(*a, **ka)
2018-09-23 22:21:36	stdout	  File "/app/bazarr.py", line 271, in series
2018-09-23 22:21:36	stdout	    c.execute("SELECT COUNT(*) FROM table_shows")
2018-09-23 22:21:36	stdout	OperationalError: database is locked
```

UI is no more responding, and I sometime get the 500 error page, for example:
```
Traceback (most recent call last):
  File "/app/libs/bottle.py", line 862, in _handle
    return route.call(**args)
  File "/app/libs/bottle.py", line 1740, in wrapper
    rv = callback(*a, **ka)
  File "/app/bazarr.py", line 139, in wrapper
    return func(*a, **ka)
  File "/app/bazarr.py", line 290, in series
    output = template('series', __file__=__file__, bazarr_version=bazarr_version, rows=data, missing_subtitles_list=missing_subtitles_list, total_subtitles_list=total_subtitles_list, languages=languages, missing_count=missing_count, page=page, max_page=max_page, base_url=base_url, single_language=single_language, page_size=page_size)
  File "/app/libs/bottle.py", line 3619, in template
    return TEMPLATES[tplid].render(kwargs)
  File "/app/libs/bottle.py", line 3409, in render
    self.execute(stdout, env)
  File "/app/libs/bottle.py", line 3396, in execute
    eval(self.co, env)
  File "/app/views/series.tpl", line 49, in <module>
    % include('menu.tpl')
  File "/app/libs/bottle.py", line 3386, in _include
    return self.cache[_name].execute(env['_stdout'], env)
  File "/app/libs/bottle.py", line 3396, in execute
    eval(self.co, env)
  File "/app/views/menu.tpl", line 38, in <module>
    % wanted_series = c.execute("SELECT COUNT(*) FROM table_episodes WHERE missing_subtitles != '[]'" + monitored_only_query_string).fetchone()
OperationalError: database is locked
```

Restart does not change. Only solution: kill the database and restart (and reconfigure subtitles downloaders and notif). Do you know how to unlock this database?

**Comment by morpheus65535** [comment]
Working on this issue. Thanks for your patience.

**Comment by gsemet** [comment]
Thanks for bazarr ! I have started a similar tool ([dopplerr](https://github.com/gsemet/dopplerr)) but I did not maintained it, because happy event in the familly; so I switched to bazarr recently (from [hotio images](https://gitlab.com/hotio/suitarr)) and found it quite elegant ! Great job, if you do not mind, I'll keep raise your issues to help improve your soft.

And if you need help for dev, do not hesitate to ask me ;)

**Comment by morpheus65535** [comment]
Thanks! Don't hesitate to report issue :-)

I'll take a look at dopplerr btw.

**Comment by morpheus65535** [comment]
Should be fixed in 0.6.5.
